### ISSUE TITLE
mixture of gene symbol and gene name is confusing

### ISSUE THREAD

**Issue by nlwashington** [monarch-initiative/monarch-legacy#1179]
the monarch front-end is displaying a mixture of gene full-length names and symbols.  these should be standardized.  i'd recommend on symbol.  @jnguyenx can you figure out what source is contributing the long label?  and then when propagating the labels for the cliqueLeader, either take as the label the synonym that is an abbreviation, or fix the data at dipper time to ensure that the symbol is the name.  

somewhat related to https://github.com/monarch-initiative/dipper/issues/59

**Comment by jnguyenx** [comment]
Can you please provide an example?

As far as I know, in SciGraph we have only one label which is the http://www.w3.org/2000/01/rdf-schema#label rdf property.

**Comment by cmungall** [comment]
yep, the issue is that some dipper sources may be putting the wrong thing into the rdfs:label. 

An example:
http://monarchinitiative.org/gene/NCBIGene:4137

One of the following has the erroneously long label:
ENSEMBL:ENSG00000186868, OMIM:157140, HGNC:6893, Orphanet:123144, KEGG-hsa:4137

**Comment by nlwashington** [comment]
I fear it may be OMIM...start there.

**Comment by kshefchek** [comment]
@cmungall was this an [attempt](https://github.com/monarch-initiative/dipper/commit/09551878c40c92f3943fb41325afe05db4da7e0c) to fix?  I'm unable to run OMIM until we work out licensing.
